src/types.js

```diff
@@ -0,0 +1,8 @@
+import PropTypes from 'prop-types';
+
+export const MovieCardTypes = PropTypes.shape({
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
```

Comment:
According to the task, `description` is not required so you should just set a default value for it